cppcache/src/statistics/StatisticsTypeImpl.cpp

```diff
@@ -45,11 +43,11 @@ StatisticsTypeImpl::StatisticsTypeImpl(std::string nameArg,
   }
   if (statsLengthArg > MAX_DESCRIPTORS_PER_TYPE) {
     char buffer[100];
-    ACE_OS::snprintf(buffer, 100, "%d", statsLengthArg);
+    std::snprintf(buffer, 100, "%d", statsLengthArg);
     std::string temp(buffer);
     std::string s = "The requested descriptor count " + temp +
                     " exceeds the maximum which is ";
-    ACE_OS::snprintf(buffer, 100, "%d", MAX_DESCRIPTORS_PER_TYPE);
+    std::snprintf(buffer, 100, "%d", MAX_DESCRIPTORS_PER_TYPE);
     std::string buf(buffer);
     s += buf + ".";
     throw IllegalArgumentException(s.c_str());
```

Comment:
This looks a little crazy.  Can't we just include the "." in the snprintf call?   And doesn't IllegalArgumentException have a ctor that takes a std::string?  So we could just get rid of some of the work and `throw IllegalArgumentException(s + buf);`?

Reply:
This was a global search and replace. I will clean up this though.